components/compass_theme_provider/compass_theme_provider.tsx

```diff
@@ -20,6 +87,135 @@ const CompassThemeProvider = ({theme, children}: Props): JSX.Element | null => {
         noFontFaces: true,
     });
 
+    //     if (!UserAgent.isFirefox() && !UserAgent.isInternetExplorer() && !UserAgent.isEdge()) {
```

Comment:
Same here

Reply:
that is on purpose as well to see if it adds a regression or not

Reply:
Can we add anything to make sure we remove this piece of code if we don't find any regression? (i.e. before we merge)

Reply:
I did move this specific rule to the `_global.scss` file